src/main/java/net/liquidpineapple/pang/screens/LevelIterator.java

```diff
@@ -83,7 +84,7 @@ public int compare(LevelScreen level1, LevelScreen level2) {
   private void loadArrayIde(final String directory) {
     //get the files from the directory
     File[] files = new File(Application.class.getResource("/" + directory).getFile()).listFiles();
-    Pattern pattern = Pattern.compile("levels\\\\(single|multi)player\\\\level\\d*.xml");
+    Pattern pattern = Pattern.compile("levels\\\\(single|multi|custom)player\\\\level\\d*.xml");
```

Comment:
customplayer? Wouldn't something like "levels\\\\((?:single|multi)player|custom)\\\\level\\d*.xml" make more sense?
That would match singleplayer/multiplayer/custom instead of singleplayer/multiplayer/customplayer.